events/wc_crisis_events.txt

```diff
@@ -0,0 +1,758 @@
+			}
+		}
+		# Fallback
+		else = {
+			create_character = {
+				random_traits = yes
+				dynasty = none
+				religion = tidemother
+				culture = tirassian
+				female = yes
+				age = 26
+				attributes = {
+					martial = 7
+				}
+				trait = creature_human
+				trait = brilliant_strategist
+				trait = class_paladin_7
+				trait = zealous
+			}
+			new_character = {
+				save_event_target_as = target_ooe_founder	# Saves new target
+			}
+		}
+		event_target:target_ooe_founder = { narrative_event = { id = WCCRS.120 } }	# Creates the Order
+	}
+}
+# Delayed event
+narrative_event = {
+	id = WCCRS.120
+	title = EVTTITLE_WCCRS_120
+	desc = EVTDESC_WCCRS_120
+	picture = GFX_evt_drustvar
+	border = GFX_event_narrative_frame_war
+
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+
+	is_triggered_only = yes
+
+	immediate = {
+		log = "WCCRS.120 fired for [Root.GetBestName]"
+		
+		clr_global_flag = drustvar_crisis_ongoing_flag
+
+		religion = tidemother	# This's tidemother's order
+		clear_secret_religion = yes
+		add_trait = zealous
+
+		activate_title = { title = d_order_of_embers status = yes }
+		d_order_of_embers = { grant_title = event_target:target_ooe_founder }
+		if = {
+			limit = {
+				k_kul_tiras = {
+					holder_scope = {
+						is_dark_being_trigger = no
+						religion = tidemother
+					}
+				}
+			}
+			set_defacto_liege = k_kul_tiras
+		}
+		religion_authority = {
+			modifier = religious_order_formed
+			years = 10
+		}
+		event_target:target_ooe_founder = {
+			wealth = 500
+		}
+		2193 = {
+			ROOT = {
+				spawn_unit = {
+					owner = ROOT
+					province = PREV
+					home = PREV
+					troops = {
+						archers = { 1000 1000 }
+						light_infantry = { 1500 1500 }
+						heavy_infantry = { 1875 1875 }
+						light_cavalry = { 250 250 }
+						knights = { 375 375 }
+					}
+					cannot_inherit = no
+					attrition = 1.0
+					reinforces = yes
+					maintenance = no
+				}
+			}
+		}
+		# Declares war against the duchy of Drustvar
+		d_drustwar = {
+			holder_scope = {
+				if = {
+					limit = {
+						OR = {
+							character = event_target:target_worried_spouse
+							character = event_target:target_sick_spouse
+						}
+					}
+					reverse_unsafe_war = {
+						target = event_target:target_ooe_founder
+						casus_belli = religious
+						thirdparty_title = PREV
+					}
+				}
+			}
+		}
+	}
+
+	option = {
+		trigger = {
+			true_religion = throsic
+		}
+		name = EVTOPTA_WCCRS_120
+	}
+	option = {
+		trigger = {
+			NOT = {
+				true_religion = throsic
+			}
+		}
+		name = EVTOPTB_WCCRS_120
+	}
+}
+
+#Gorak Tul invades - triggered by decision
+narrative_event = {
+	id = WCCRS.130
+	title = EVTTITLE_WCCRS_130
+	desc = EVTDESC_WCCRS_130
+	picture = GFX_evt_drustvar
+	border = GFX_event_narrative_frame_intrigue
+
+	is_triggered_only = yes
+
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+
+	immediate = {
+		log = "WCCRS.130 fired for [Root.GetBestName]"
+		
+		set_global_flag = drust_incursion_happened
```

Comment:
Thinking about it. The sick spouse events shouldn't happen if the drusts are already on Azeroth, right?

Reply:
If the crisis fired once or someone summoned the Drust warband by somehow gaining Throsic religion and conquering duchy of Drustvar without the spouse events happening, then they should be locked.

Reply:
They're not now.